website/blog/2023-4-10-incrementals-pt1.md

```diff
@@ -0,0 +1,140 @@
+`onedriveMetadata` instead of `onedrive`) to make it straightforward to separate out from
+backed up item data.
+
+### Terminology
+
+*Full backups* are backups where all of the data being backed up is fetched from
+M365 with the Graph API. These backups may take a long time to complete (we’ve
+seen backups that run for 20+ hours) due to throttling imposed by Microsoft 365.
+For the purposes of this blog, *incremental backups* are backups where Corso
+fetches only a subset of items from M365. Ideally Corso would fetch only the
+items that change, though there may be reasons it needs to fetch more data.
+
+Whether Corso does a full backup or an incremental backup, the resulting Corso
+backup has a listing of all items stored in M365 (what we refer to as *indexing
+information*). This means there’s no “chaining” between backups and restoring an
+item from a backup requires only accessing information contained in or
+referenced directly by the backup passed in to the restore command. This makes
+backups independent from each other once they’ve been created, so we’ll refer to
+them as *independent backups* for the rest of this post.
+
+Both independent backups and chained backups have the same information. Having
+independent backups generally creates more complexity when making a new backup
+while chained backups generally have more complexity during restore and backup
+deletion. Independent backups have more complexity when creating the backup as
+indexing information and item data references for deduplicated data may need to
+be sourced from previous backups. Chained backups have more complex restore as
+multiple backups may need to be searched for the item being restored. They also
+have more complex backup deletion as an item’s data can only be deleted if no
+backups in any chain refer to it. The figure below gives a high-level overview
+of the differences between independent backups and chained backups.
+
+![an image of an independent backup](./images/independent_backups.png)
+*both images below show how data would be stored if the user backed up two files on their first backup and then made a*
+*new file and updated file1 before taking a second backup*
+![an image of a chained backup](./images/chained_backups.png)
+
+Although having a full listing of all items present at the time of the backup in
+each backups sounds wasteful, Corso takes advantage of the data deduplication
+provided by kopia to only store one copy of the underlying data for backed up
+items. What this really means is each Corso backup has a complete set of
+*indexing information*. This gives Corso the best of both worlds; allowing
+completed backups to have independent indexing information and life cycles from
+each other while still minimizing the amount of item data stored.
+
+> 💡 In [part 2 of our series, we’ll cover Incremental backups in action](https://corsobackup.io/blog/incremental-backups-pt2/).
```

Comment:
This should be longer but, in general, needs a better summary that describes what we covered in this blog post, what is coming next and a link to where to find it.

Reply:
expanded the conclusion